index.html

```diff
@@ -206,7 +206,7 @@ <h3>
             </dd>
           </dl>
           <dl>
-            <dt id="linked-data-integrity-1" class="spec">Verifiable Credential Linked Data Integrity 1.0</dt>
+            <dt id="linked-data-integrity-1" class="spec">Linked Data Integrity 1.0</dt>
```

Comment:
Adding "Verifiable Credential" to the name of this deliverable was _strongly_ recommended by Ivan to forestall some of the issues run into by the linked data signatures charter.

Reply:
I'm afraid that the name might cause people in the WG to hard code the solution to VCs, when the solution is intended to be generalized. I won't stand in the way here, but will raise the concern that people might take the wrong thing away from the title of the specification. I can live with it for now, and hope it won't blow up in our faces later. :)

Reply:
To make my point clear: I do not want to see the same discussion coming up again from the LD community. If other communities pick up the VC solution for other problems than VC-s, good for them and for us. But if we just make hint that we have a solution for Linked Data at large we are back to square one. Other communities may vote with their feet, a.k.a. their code. It is not this wg's concern.

B.t.w., we should be consistent: in the other PR the deliverable's name in one of the liaison statement was also using the title of the spec without VC.

(we shoukd add the preview facility to this repo. I have difficulties to look PR-s on my iPad...)